### DISCUSSION TITLE
Directory for custom PBF file path

### DISCUSSION THREAD

**Discussion by AppIe314** [mediagis/nominatim-docker#377] [Q&A] [answered]
I have created a custom PBF file by merging two other PBF files. I am now trying to re-build nominatim-docker with this new PBF file. From the docker documentation I can use this command:
`docker run -it \
  -e PBF_PATH=/nominatim/data/merged.osm.pbf \
  -p 8080:8080 \
  -v /osm-maps/data:/nominatim/data \
  --name nominatim \
  mediagis/nominatim:4.0`

My issue is that I don't know where to place the merged pbf file. I can't find /nominatim/data/.
I have tried putting it in the docker folder. For me this is home/usernamehere/nominatim-docker/4.0/merged.pbf
However I am getting a message to say the OSM file does not exist followed by "FATAL: Cannot access file".

I imagine this is an easy fix but docker is truly magic to me.

**Comment by leonardehrenfried** [comment, marked as answer]
You need to use the correct volume mapping. The bit before the colon is on your host computer that is mounted into the container at the path after the colon.

So in your case it's

```
docker run -it \
  -e PBF_PATH=/nominatim/data/merged.osm.pbf \
  -p 8080:8080 \
  -v /home/usernamehere/nominatim-docker/4.0:/nominatim/data \
  --name nominatim \
  mediagis/nominatim:4.1
```

**Reply by AppIe314** [reply]
Oh I see! I was trying to change the PBF_PATH env variable.
Thank you so much for your rapid response. That works perfectly.
